src/renderers/webgl/WebGLTextures.js

```diff
@@ -753,14 +775,14 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 	// Render targets
 
 	// Setup storage for target texture and bind it to correct framebuffer
-	function setupFrameBufferTexture( framebuffer, renderTarget, attachment, textureTarget ) {
+	function setupFrameBufferTexture( framebuffer, renderTarget, texture, attachment, textureTarget ) {
 
 		var glFormat = utils.convert( renderTarget.texture.format );
 		var glType = utils.convert( renderTarget.texture.type );
 		var glInternalFormat = getInternalFormat( glFormat, glType );
 		state.texImage2D( textureTarget, 0, glInternalFormat, renderTarget.width, renderTarget.height, 0, glFormat, glType, null );
```

Comment:
This can probably way but is it possible for the different color attachments to have different sizes?

Reply:
I am not sure if different image sizes have any real-world usage (if supported all). Although the fragment stage outputs several colors, the rasterization stage has been executed only once, so separate target resolutions are not possible. To write to the smaller/bigger images the driver would need to down/up sample which might be possible but the gain of having some of them smaller is probably none. The gain of MRT is to exactly execute everything prior the fragment shader once only.

According to this post it might work but will be rendered with the size of the smallest image https://community.khronos.org/t/fbo-with-different-size-attachments/62925/2

P.S. Different formats and types for the images are possible and have practical usage. I read that some combination could possibly fail depending on the implementation. Not sure if this applies to WebGL or OpenGL only.

Reply:
I'd like to go as is so far, and consider different size if users actually demand it.